planning/autoware_route_handler/include/autoware/route_handler/route_handler.hpp

```diff
@@ -353,17 +355,6 @@ class RouteHandler
   lanelet::ConstLanelets getMainLanelets(const lanelet::ConstLanelets & path_lanelets) const;
 
   // for lanelet
-  bool isBijectiveConnection(
-    const lanelet::ConstLanelets & lanelet_section1,
-    const lanelet::ConstLanelets & lanelet_section2) const;
-  bool getPreviousLaneletWithinRouteExceptGoal(
-    const lanelet::ConstLanelet & lanelet, lanelet::ConstLanelet * prev_lanelet) const;
-  bool getNextLaneletWithinRouteExceptStart(
-    const lanelet::ConstLanelet & lanelet, lanelet::ConstLanelet * next_lanelet) const;
-  bool getRightLaneletWithinRoute(
-    const lanelet::ConstLanelet & lanelet, lanelet::ConstLanelet * right_lanelet) const;
-  bool getLeftLaneletWithinRoute(
-    const lanelet::ConstLanelet & lanelet, lanelet::ConstLanelet * left_lanelet) const;
```

Comment:
Is there any reason why these unused code  is removed in the unit test's PR?

Reply:
I remember I saw somewhere in the testing guidelines to remove unused member functions ...